services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_scheduler.py

```diff
@@ -54,11 +54,6 @@ async def update_service_observation(
     summary="Removes the service's user services",
     status_code=status.HTTP_202_ACCEPTED,
     response_model=TaskId,
```

Comment:
so just in case this is used somewhere else as well...

Why not just catch the exception and return the task instead of preventing it from happening?
From the long running task framework perspective I think I preferred the old way.

Reply:
Yes, I don't know why I did not go for it. But now it's refactored like that